src/components/SoundBox/ColumnOfBoxes/ColumnOfBoxes.test.js

```diff
@@ -8,4 +8,12 @@ describe('<ColumnOfBoxes />', () => {
     const renderedColumnOfBoxes = mount(<ColumnOfBoxes color="red" />);
     expect(renderedColumnOfBoxes.find(SimpleBox)).toHaveLength(4);
   });
+
+  it('ColumnOfBoxes changes state', () => {
```

Comment:
What do you think if we provide more clarification and cover this case more detail with describe and write at least two test that. For example when we click again on the same box it becomes disabled.